### ISSUE TITLE
UI bug: Missing links

### ISSUE THREAD

**Issue by tinybigideas** [argoapp-live/beta-testing#21]
## Describe the bug

Links on billing page are not wired up.

## Criticality

Low?

## To Reproduce

1. Go to https://aqua.spheron.network/#/billing/plan

## Expected behavior

Links are wired up to go where you'd expect

## Screenshots

![image](https://user-images.githubusercontent.com/2805752/164912617-f9752d94-5840-426d-aebe-26b1df6631bb.png)

## Your Details

- Web3 Name: tinybigideas
- Twitter handle: tinybigideas

**Comment by rekpero** [comment]
Yep, these links are broken, will be fixed in our upcoming release. 

Thanks for reporting @tinybigideas
